R/helper-functions.R

```diff
@@ -126,6 +131,13 @@ validate_color <- function(color = NULL, or_null = FALSE, envir = parent.frame()
   # remove any # from color strings
   color <- gsub("^#", "", toupper(color))
 
+  # if the format is RGBA (R's default), switch first two with last two characters
+  if (format == "RGBA") {
+    s <- nchar(color) == 8
+    alpha <- substring(color[s], 7, 8)
+    color[s] <- paste0(alpha, substring(color[s], 1, 6))
+    }
```

Comment:
Please move this to the beginning of the line

Reply:
`}` moved two spaces left